LeaderboardBackend.Test/Controllers/LeaderboardsControllerTest.cs

```diff
@@ -0,0 +1,72 @@
+
+using NUnit.Framework;
+using Moq;
+using LeaderboardBackend.Controllers;
+using LeaderboardBackend.Services;
+using LeaderboardBackend.Models;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+
+namespace LeaderboardBackend.Test.Controllers;
+
+public class LeaderboardTests
+{
+	private Mock<ILeaderboardService> _leaderboardServiceMock = null!;
+
+	private LeaderboardsController _controller = null!;
+
+    [SetUp]
+    public void Setup()
+    {
+		_leaderboardServiceMock = new Mock<ILeaderboardService>();
+
+		_controller = new LeaderboardsController(
+			_leaderboardServiceMock.Object
+		);
+    }
+
+    [Test]
+    public async Task TestGetLeaderboardNotFoundIfLeaderboardNotFound()
```

Comment:
[This test naming style](https://github.com/leaderboardsgg/leaderboard-backend-poc/blob/main/LeaderboardBackend.Test/Controllers/UsersControllerTest.cs#L46) is what I tried in the UsersController tests, and it seems pretty readable. Would it be okay to swap these over to that or is this way better?

Reply:
Let's go with that style; definitely more readable.